<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>221</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="set_pushButton">
       <property name="text">
        <string>Stroke Library Setting</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="input_textEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="generate_pushButton">
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="result_graphicsView">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>361</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="chars_listView">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>181</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="stroke_graphicsView">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>380</y>
      <width>281</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="strokes_listView">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>381</y>
      <width>181</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Stroke Information:</string>
    </property>
   </widget>
   <widget class="QLabel" name="stroke_info_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>311</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>883</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
